# Only needed for version 3.10.45. Removes a spurious warning.
--- amrfinder.cpp	2022-10-25 04:07:28.000000000 +1300
+++ amrfinder.cpp	2022-11-16 13:48:10.524604649 +1300
@@ -534,2 +533,0 @@
-        const Warning warning (stderr);
-        stderr << "This was compiled for running under bioconda, but $CONDA_PREFIX was not found" << '\n';
@@ -537 +534,0 @@
-        stderr << "Reverting to $PREFIX: " << defaultDb;
@@ -539,3 +535,0 @@
-        const Warning warning (stderr);
-        stderr << "This was compiled for running under bioconda, but $CONDA_PREFIX was not found" << '\n';
-        stderr << "Reverting to hard coded directory: " << CONDA_DB_DIR "/latest";
