--- apptainer/internal/pkg/buildcfg/confgen/gen.go.orig	2023-02-06 18:25:11.309412000 +1300
+++ apptainer/internal/pkg/buildcfg/confgen/gen.go	2023-02-06 18:26:06.163244000 +1300
@@ -56,7 +56,7 @@
 		"APPTAINER_CONFDIR",
 		"PLUGIN_ROOTDIR":
 		varType = "var"
-		varStatement = d.Words[1] + " = relocatePath(" + s + ")"
+		varStatement = d.Words[1] + " = " + s
 	case "APPTAINER_SUID_INSTALL":
 		varType = "var"
 		varStatement = d.Words[1] + " = isSuidInstall()"
