--- jupyter_core/paths.py.orig	2023-01-20 13:43:16.377554000 +1300
+++ jupyter_core/paths.py	2023-01-20 13:44:18.470052082 +1300
@@ -224,7 +224,7 @@
             "/usr/share/jupyter",
         ]
 
-ENV_JUPYTER_PATH: List[str] = [os.path.join(sys.prefix, "share", "jupyter")]
+ENV_JUPYTER_PATH: List[str] = [os.path.join(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'share', 'jupyter')]
 
 
 def jupyter_path(*subdirs: str) -> List[str]:
@@ -305,7 +305,7 @@
             "/usr/local/etc/jupyter",
             "/etc/jupyter",
         ]
-ENV_CONFIG_PATH: List[str] = [os.path.join(sys.prefix, "etc", "jupyter")]
+ENV_CONFIG_PATH: List[str] = [os.path.join(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'etc', 'jupyter')]
 
 
 def jupyter_config_path() -> List[str]:
