--- jupyterlab/commands.py.orig	2021-04-12 16:26:30.335708000 +1200
+++ jupyterlab/commands.py	2021-04-12 16:27:12.417708981 +1200
@@ -152,7 +152,7 @@
         return str(Path(os.environ['JUPYTERLAB_DIR']).resolve())
 
     # Use the default locations for data_files.
-    app_dir = pjoin(sys.prefix, 'share', 'jupyter', 'lab')
+    app_dir = pjoin(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'share', 'jupyter', 'lab')
 
     # Check for a user level install.
     # Ensure that USER_BASE is defined
