--- jupyter_core/paths.py.orig	2021-10-28 10:14:38.399794000 +1300
+++ jupyter_core/paths.py	2021-10-28 10:15:46.974414511 +1300
@@ -131,7 +131,7 @@
         "/usr/share/jupyter",
     ]
 
-ENV_JUPYTER_PATH = [os.path.join(sys.prefix, 'share', 'jupyter')]
+ENV_JUPYTER_PATH = [os.path.join(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'share', 'jupyter')]
 
 
 def jupyter_path(*subdirs):
@@ -202,7 +202,7 @@
         "/etc/jupyter",
     ]
 
-ENV_CONFIG_PATH = [os.path.join(sys.prefix, 'etc', 'jupyter')]
+ENV_CONFIG_PATH = [os.path.join(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'etc', 'jupyter')]
 
 
 def jupyter_config_path():
